app.js

```diff
@@ -75,7 +83,7 @@ function updateChoice() {
     // put the new displayed array into local storage
     addLocalDisplay(allDisplayed);
     // after this many inputs change to results page
-    if (storedPicks.length >= 25) {
+    if (storedPicks.length >= 3) {
```

Comment:
You may want to change this back...it's redirecting on the first pick.

Reply:
oh yeah I did that so I didn't have to click through 25 times when testing things haha